### PR TITLE
Add Windows compatibility to script

### PR DESCRIPTION
**Pull request by DetermineAbsurd**
I took the liberty of adding compatibility for Windows since I saw that the original file was hardcoded to work with Linux.

EDIT: Also added some print statements so the end user doesn't think the script is hanging.

### CODE REVIEW

File: jellyfin-collection-fix.py
Comment (oddstr13) [inline jellyfin-collection-fix.py]: I *strongly* disagree with this PEP8 convention.
Functions are `camelCase`. Classes are `PascalCase`. Variables are `snake_case`. Constants are `SNAKE_CAPS`.

File: jellyfin-collection-fix.py
Comment (oddstr13) [inline jellyfin-collection-fix.py:17]: This is missing msys2, I don't know exactly what the variable would be.

File: jellyfin-collection-fix.py
Comment (oddstr13) [inline jellyfin-collection-fix.py]: This is a good place for a multiline string. Or multiple prints.
Separate the printed junk from the actual prompt.
Display the default to the user.

File: jellyfin-collection-fix.py
Comment (oddstr13) [inline jellyfin-collection-fix.py]: `os.path.join()`. *Allways* use it when joining filesystem paths.

File: jellyfin-collection-fix.py
Comment (oddstr13) [inline jellyfin-collection-fix.py]: Don't change coding style in a functional pull-request.
`str.format()` is perfectly fine.

File: jellyfin-collection-fix.py
Comment (oddstr13) [inline jellyfin-collection-fix.py]: There is *absolutely* no reason to split this string over multiple lines.
Entered path should also be verified where it is entered (`os.path.exists()`), and not rely on the write access check.

File: jellyfin-collection-fix.py
Comment (oddstr13) [inline jellyfin-collection-fix.py]: I don't think this print is needed, as the script will exit otherwise.
Status updates before potentially long-running blocks of code are fine.

File: jellyfin-collection-fix.py
Comment (oddstr13) [inline jellyfin-collection-fix.py]: I get it, this line is long.
But that does not excuse the poor choice of where the lines are split, and this change does not improve code readability.

If anything, this should probably be a multiline constant somewhere near the beginning of the file.
(I'm assuming multiline strings would be fine for a single sqlite statement).

Either leave these lines alone, or format them to improve SQL readability.

File: jellyfin-collection-fix.py
Comment (oddstr13) [inline jellyfin-collection-fix.py]: If you have to break up this line, use JSON style indented dictionary.
```python
..., {
    'eids': '|'.join(new_eids),
    'guid': guid,
})
```

File: jellyfin-collection-fix.py
Comment (oddstr13) [inline jellyfin-collection-fix.py]: eww, why is this even valid syntax?

File: jellyfin-collection-fix.py
Comment (oddstr13) [inline jellyfin-collection-fix.py]: See above.

**Review by oddstr13 — CHANGES_REQUESTED** [review]
In general, keep code style and formatting changes out of functional pull-requests, and follow the code style present in the existing code.

**Comment by sparky8251** [comment]
If you can fix the style concerns and clean up the code as @oddstr13 has suggested we would be happy to accept this patch.

We just cannot accept patches that make scripts into odd hybrid styles and where possible we try to stick to the languages style.

File: jellyfin-collection-fix.py
Comment (DetermineAbsurd) [reply jellyfin-collection-fix.py]: Are you referring to the import, the f string or both?

Comment (oddstr13) [reply jellyfin-collection-fix.py]: import is fine - pretty much standard way of doing platform-specific imports.

**Comment by oddstr13** [comment]
I hope this doesn't sound too harsh, but I'm leaning towards keeping this PR open as an issue, and implementing the changes myself when I can test it. Both because of all the unrelated changes in this PR, and because I'd want to test the changes before merging.
A way to detect running Jellyfin on Windows should really be implemented too, unless we can just fall back on Windows' tendency to lock all them files.

I'll probably be looking into setting up a Windows VM for testing this and other things this week.

Don't be too discouraged tho, as I'm happy to have issues pointed out, and suggestions of how to fix it / proper patches.
It's just that I'm not a fan of code style changes, unless there are *really* good reasons for it, and most if not all such changes in this PR are just noise in the diff.

**Comment by DetermineAbsurd** [comment]
Fair enough. I'll revert the requested styling changes back to original and change those other things as requested. The implementation of detecting if Jellyfin is running on Windows is outside of my knowledge of Python, so that is my reasoning for not implementing it. I found solutions from various stackoverflow threads, but the majority of them were outdated or required third party libraries.

**Review by oddstr13 — APPROVED** [review]

**Pull request merged**
